src/cmd-kola

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+dn=$(dirname $0)
+. ${dn}/cmdlib.sh
+
+latest_qcow=$(get_latest_qcow)
+if [ -z "${latest_qcow}" ]; then
+    fatal "No latest build"
+fi
+
+exec sudo kola -b rhcos --tapfile test.tap --qemu-image ${latest_qcow} "$@"
```

Comment:
is `-b rhcos` important here? is there a `-b fcos` ?

Reply:
Distribution flags are important. That said, we should make it a flag on the command and just have it default to one of fcos/rhcos

Reply:
I'd vote for changing kola to look for `rhcos-` or `fcos-` as a prefix in the image name for now; that same thing would also be useful for ignition spec version handling.

Reply:
Prefix based detection WFM.

And yes, when `--ignition-version v2` will need to be specified when running RHCOS until spec 3.0.0 lands there. For FCOS we don't need to specify that parameter as it defaults to v3 on that distribution.

Reply:
Would probably be good to have command-line overrides as well (at least on Ignition version) so that it would be possible to test Ignition spec 3.0.0 in RHCOS without having to modify `cmd-run`.

Reply:
I added the auto-detect logic in the command itself for now with a `XXX` to upstream it into kola.